jenkins-slaves/ansible/kie-rhel7.yml

```diff
@@ -22,6 +22,7 @@
     - common
     - kie
     - rhba
+    - rhba-osbs
```

Comment:
Was it forgotten to add  that role here?  I can't see it in the directory ;) it is empty.

Reply:
No, I did it on purpose :) The role is stubbed out in the public repo. You have to run "add-osbs.sh <url of bxms-jenkins repo>" to fill in the role

Reply:
It's noted in the README

Reply:
i see, ok i got it now